testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -1302,8 +1302,8 @@ def test_get_velocities(self):
         v = self.ag.get_velocities()
         assert_(np.any(np.abs(v) > 1e-6), "velocities should be non-zero")
 
-    def test_vel_src(self):
-        assert_equal(self.universe.trajectory.ts.data['velocity_source'], 0)
+    # def test_vel_src(self):
```

Comment:
It's for TRR to keep track of which frame data came from.

https://github.com/MDAnalysis/mdanalysis/blob/develop/package/MDAnalysis/coordinates/xdrfile/TRR.py#L135

So if positions are set on frame 1, then a new frame gets read that didn't have positions, the data becomes "stale" and the Timestep acts like it has no data